sql/moz-fx-data-shared-prod/telemetry_derived/cohort_weekly_active_clients_v1/query.sql

```diff
@@ -0,0 +1,14 @@
+SELECT DISTINCT
+  client_id,
+  DATE_TRUNC(submission_date, WEEK) AS activity_date_week
+FROM
+  `moz-fx-data-shared-prod.telemetry_derived.cohort_weekly_active_clients_staging_v1`
+WHERE
+  submission_date = DATE_TRUNC(
+    DATE_SUB(@submission_date, INTERVAL 768 day),
+    WEEK
+  ) --start of week for date 768 days ago
```

Comment:
This should be fine since `cohort_weekly_active_clients_staging_v1` is probably small but can't this be incremental as well?  Since the active clients in past weeks isn't going to change